### DISCUSSION TITLE
Where are stable (non-beta) releases?

### DISCUSSION THREAD

**Discussion by WTPascoe** [PowerShell/Win32-OpenSSH#2089] [Q&A]
This is probably a really dumb question, but where can I find non-beta releases? Looking at https://github.com/PowerShell/Win32-OpenSSH/releases/latest/ shows a beta version and looking through https://github.com/PowerShell/Win32-OpenSSH/releases, I can't find a 8.9 non-beta package

**Comment by maertendMSFT** [comment]
This was answered in the issue that you opened: https://github.com/PowerShell/Win32-OpenSSH/issues/2090

We will update the docs/readme.
